### PR TITLE
Add Chain Letter by MIT Technology Review

### PR DESCRIPTION
**Pull request by raulra08**
All other newsletters can be found in https://forms.technologyreview.com/newsletters/

### CODE REVIEW

**Comment by ferhatelmas** [comment]
@raulra08 thank you for all these changes. Could you rebase ?

**Comment by raulra08** [comment]
Hi @ferhatelmas,
Sure, let me try to do that now

**Comment by raulra08** [comment]
@ferhatelmas 
The conflict should be fixed now
Thanks for letting me know ✌🏼

**Comment by raulra08** [comment]
Nice! 💪🏼
The build is back on track!
I kind of feel sorry I removed so many ^^
But many are actually not working anymore, some don't have a newsletter there anymore and other websites have actually gone http**s** which is pretty good.

**Comment by ferhatelmas** [comment]
Awesome.

Just one comment, why did you remove `unreadit` newsletters?

**Comment by raulra08** [comment]
Initially the tests couldn't connect to the '.it' domain so I updated the url to the .com version as suggested by the test.
Today I saw `unreadit` failed to connect to the '.com' domian.
So I manually checked what's going on today and site seems to be down as shown below:

Screenshot 2020-04-06 at 21 57 57

**Comment by ferhatelmas** [comment]
Correct one is https://unreadit.com/n/frontend/
It seems you mistyped url. When you add them back, we are good to go.

**Comment by raulra08** [comment]
Done; I have added Unreadit back 👍

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 - [Front End Newsletter](http://frontendnewsletter.com/). The free, weekly newsletter of the best Front End related news, articles, projects, and more. JavaScript, React, Angular, Ember, Polymer, Meteor, RxJS, Elm, and more. [Archive](http://frontendnewsletter.com/issues).
 - [Dev Tips](https://umaar.com/dev-tips/). A developer tip, in the form of a gif, in your inbox each week.
 - [Friday Front-end](https://zendev.com/friday-frontend.html). 15 great Front-end articles, tutorials, and announcements every Friday. Sections for CSS/SCSS, JavaScript, and other awesome stuff. [Archive](https://zendev.com/category/friday-frontend.html)
-- [Unreadit/Frontend](https://unread.it/n/frontend/). Best frontend development news sourced from the best subreddits.
+ - [Unreadit/Frontend](https://unreadit.com/n/frontend/). Best frontend development news sourced from the best subreddits.
```
Comment (ferhatelmas) [inline README.md:77]: it seems we introduce an unnecessary whitespace.

File: README.md [hunk 2]
```diff
@@ -148,11 +145,9 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
...
 - [The UI Animation Newsletter](https://us2.list-manage.com/subscribe?u=6fbaddc8c1fce7588d1a35cb2&id=8f4de2c2e5). The UI Animation Newsletter keeps you up to date on the best web animation, motion design, and UX resources.
-- [Web Animation Weekly](http://webanimationweekly.com/). A hand-picked selection of articles, videos, book reviews, and other goodies pertaining to the wonderful worlds of web animation and motion design!
+- [Web Animation Weekly](http://rachelnabors.com/). A hand-picked selection of articles, videos, book reviews, and other goodies pertaining to the wonderful worlds of web animation and motion design!
 - [Design Systems News](http://news.design.systems/). Latest news, articles, repos, code, frameworks, videos and podcasts on Design Systems, Pattern Libraries and Styleguide. [Archive](http://news.design.systems/issues).
 - [Designgest](http://designgest.com/). Designgest is an easy way to get fresh information about interactive design trends.
-- [Unreadit/Design](https://unread.it/n/design/). Design inspiration and great design-related content sourced from the best subreddits.
```
Comment (ferhatelmas) [inline README.md:154]: is this gone too or forgotten to revert ?

File: README.md [hunk 3]
```diff
@@ -377,16 +368,16 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
...
 
 ## Blockchain / Cryptocurrencies
 
-- [Inside Bitcoin](https://inside.com/bitcoin). Tracking trends, news, and analysis around Bitcoin and cryptocurrencies.
+- [Inside Bitcoin](https://inside.com/cryptocurrency/?). Tracking trends, news, and analysis around Bitcoin and cryptocurrencies.
 - [Crypto Weekly](https://cryptoweekly.co/). The best cryptocurrency news and insights delivered to your inbox every week.
 - [Build Blockchain](https://www.buildblockchain.tech/newsletter). Blockchain tech without the bull— a weekly dose of blockchain reality with a heavy bias toward the technical.
+- [Chain Letter](https://technologyreview.us11.list-manage.com/subscribe?u=47c1a9cec9749a8f8cbc83e78&id=e2349bbf6b). MIT Technology Review's The Chain Letter. The author [Mike Orcutt](https://twitter.com/mike_orcutt) writes about what’s new in the world of blockchains and cryptocurrencies and why they matter. The MIT Technology review issues several other newsletters
```
Comment (ferhatelmas) [inline README.md:380]: to be consistent, could you add a dot at the end?

File: README.md [hunk 4]
```diff
@@ -438,19 +428,15 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
...
 - [lkml.wtf](https://lkml.wtf/). A weekly roundup of Linux Kernel Mailing List threads but with a lot more sarcasm.
 - [The ListServe](http://thelistserve.com/). A free, unique "lottery" newsletter where every day a subscriber is picked whose email submission is sent to all of the other subscribers. Subject matter varies in both topic and quality.
-- [Engineering Impact](https://www.gitprime.com/engineering-impact/). Weekly newsletter for managers of engineering teams covering leadership, culture, scaling, and software development productivity.
 - [tl;dr Marketing](https://tldrmarketing.com/). Weekly newsletter for digital marketers, covering news on SEO, PPC and Social Media.
 - [Startup Watching](https://startupwatching.com/). Curated knowledge, inspiration and background stories for startups. Delivered to your inbox every Monday and Thursday.
-- [Unreadit/Tech](https://unread.it/n/tech/). Hardware and software news for tech junkies from the best posts on Reddit.
-- [Unreadit/Entrepreneur](https://unread.it/n/entrepreneur/). Cutting edge news regarding entrepreneurship, growth, investing and startups sourced from the best Reddit posts.
```
Comment (ferhatelmas) [inline README.md:453]: how about these two?

**Comment by ferhatelmas** [comment]
Sorry but it seems there are still a few issues.

**Comment by ferhatelmas** [comment]
@raulra08 kind reminder for the last issues :hugs:

**Comment by raulra08** [comment]
Hi,
Sorry I got busy. I will look into this today.

**Comment by ferhatelmas** [comment]
@raulra08 no worries, thank you very much!

**Comment by ferhatelmas** [comment]
@raulra08 I hope I am not too annoying - another kind reminder :smile:

**Comment by raulra08** [comment]
Hi @ferhatelmas thanks for the reminder. Will check the standing to-do's

**Comment by ferhatelmas** [comment]
closing since no follow up.
